src/chess/position.h

```diff
@@ -49,9 +49,15 @@ class Position {
   // How many time the same position appeared in the game before.
   int GetRepetitions() const { return repetitions_; }
 
+  // How many half-moves since the same position appeared in the game before.
+  int GetPliesSinceFirstRepetition() const { return cycle_length_; }
```

Comment:
This is probably my fault, but I think `GetPliesSincePrevRepetition` is a more accurate name.

Reply:
Renamed it. Thanks :)